packages/eslint-plugin-gestalt/src/no-dangerous-style-duplicates.test.js

```diff
@@ -389,5 +396,16 @@ ruleTester.run('no-dangerous-style-duplicates', rule, {
         },
       ],
     },
+    {
+      code: invalidBoxShadow,
+      parserOptions,
+      errors: [
+        {
+          message:
+            'Un-needed Box dangerous styles found. https://gestalt.netlify.app/gestalt/#/Box\n' +
```

Comment:
I think that URL needs to be updated to remove the `/gestalt/#`

Reply:
Actually, might as well go right to the business: https://gestalt.netlify.app/Box#Borders